src/prefabs/form.tsx

```diff
@@ -63,206 +71,297 @@ const beforeCreate = ({
-                );
-                break;
-              case PropertyKind.PASSWORD:
-                structure.descendants.push(
-                  makeBettyInput(
-                    BettyPrefabs.PASSWORD,
-                    model,
-                    property,
-                    variable,
-                  ),
-                );
-                break;
-              case PropertyKind.PHONE_NUMBER:
-                structure.descendants.push(
-                  makeBettyInput(
-                    BettyPrefabs.PHONE_NUMBER,
-                    model,
-                    property,
-                    variable,
-                  ),
-                );
-                break;
-              case PropertyKind.PRICE:
-                structure.descendants.push(
-                  makeBettyInput(BettyPrefabs.PRICE, model, property, variable),
-                );
-                break;
-              case PropertyKind.URL:
-                structure.descendants.push(
-                  makeBettyInput(BettyPrefabs.URL, model, property, variable),
-                );
-                break;
-              case PropertyKind.STRING:
-                structure.descendants.push(
-                  makeBettyInput(
-                    BettyPrefabs.STRING,
-                    model,
-                    property,
-                    variable,
-                  ),
-                );
-                break;
-              case PropertyKind.TEXT:
-                structure.descendants.push(
-                  makeBettyInput(BettyPrefabs.TEXT, model, property, variable),
-                );
-                break;
-              case PropertyKind.INTEGER:
-                structure.descendants.push(
-                  makeBettyInput(
-                    BettyPrefabs.INTEGER,
-                    model,
-                    property,
-                    variable,
-                  ),
-                );
-                break;
-              case PropertyKind.BOOLEAN:
-                structure.descendants.push(
-                  makeBettyInput(
-                    BettyPrefabs.BOOLEAN,
-                    model,
-                    property,
-                    variable,
-                  ),
-                );
-                break;
-              default:
-                structure.descendants.push(
-                  makeBettyInput(
-                    BettyPrefabs.STRING,
-                    model,
-                    property,
-                    variable,
-                  ),
-                );
-            }
-            // eslint-disable-next-line no-console
-            return console.warn('PropertyKind not found');
-          });
-
+              // eslint-disable-next-line no-console
+              return console.warn('PropertyKind not found');
+            });
+          } else {
+            structure.descendants.push(cloneStructure(BettyPrefabs.STRING));
```

Comment:
What does this do?

Reply:
it will add a Textfield component to the canvas, as a sibling right before Submit button (need this in acc criteria)